src/App.jsx

```diff
@@ -39,20 +42,33 @@ function App() {
             console.log(frame)
             SocketClient.subscribe('/users/queue/messages', (message) => {
                 const messageContent = JSON.parse(message.body);
-                console.log(messageContent)
-                if (messageContent && messageContent.sender !== user.email){
-                    const notify = new Notification(messageContent.senderName, {
-                        icon: "src/assets/img/logo.svg",
-                        body: messageContent.content,
-                    });
+                // if (messageContent && messageContent.sender !== user.email) {
+                //     // const notify = new Notification(messageContent.senderName, {
+                //     //     icon: "src/assets/img/logo.svg",
+                //     //     body: messageContent.content,
+                //     // });
+                //
+                //     if (messageContent.isStatusType) {
+                //         dispatch(setTypingStatus({
+                //             user: messageContent.email,
+                //             typingStatus: messageContent.typingStatus
+                //         }))
+                //     }
+                // }
+                if (!messageContent.isStatusType) {
+                    dispatch(addUnreadMessage(messageContent))
+                    dispatch(addNewMessage(messageContent))
+                }else {
+                    dispatch(setTypingStatus({
+                        user: messageContent.sender,
+                        typingStatus: messageContent.typingStatus
+                    }))
                 }
-                dispatch(addNewMessage(messageContent))
             })
             SocketClient.subscribe('/users/queue/notify', (message) => {
-                const messageContent = message.body;
-                console.log(messageContent)
+                const notify = message.body;
```

Comment:
Using edit to notification (Noun) for variable instead of notify (Verb)

Reply:
Resolved